src/content/app/tools/shared/components/tools-app-bar/ToolsAppBar.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import noop from 'lodash/noop';
+
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import { AppName } from 'src/global/globalConfig';
+
+import { getEnabledCommittedSpecies } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
+
+import AppBar from 'src/shared/components/app-bar/AppBar';
+import { SelectedSpecies } from 'src/shared/components/selected-species';
+import SpeciesTabsWrapper from 'src/shared/components/species-tabs-wrapper/SpeciesTabsWrapper';
+
+const ToolsAppBar = () => {
+  const speciesList = useSelector(getEnabledCommittedSpecies);
+
+  const speciesTabs = speciesList.map((species, index) => (
+    <SelectedSpecies key={index} species={species} onClick={noop} />
+  ));
+  const speciesSelectorLink = useMemo(() => {
+    return <Link to={urlFor.speciesSelector()}>Change</Link>;
+  }, []);
+
+  const wrappedSpecies = (
+    <SpeciesTabsWrapper
+      isWrappable={false}
```

Comment:
Are we sure we want it to be wrappable? I thought i heard Andrea saying that it will be different for tools?

Reply:
I believe the intention was to make the list of species wrappable, i.e. that the list would grow vertically rather than squeeze species lozenges to remain on a single line. Which means that `isWrappable={false}` is incorrect; I've pushed a commit to remove it.